std/encoding.d

```diff
@@ -1355,6 +1349,7 @@ bool isValid(E)(const(E)[] s)
     return s.length == validLength(s);
 }
 
+///
 unittest
 {
     assert(isValid("\u20AC100"));
```

Comment:
Actually, nevermind... we *do* need it since this function has no examples otherwise. :-(

I'd also add a test for an *invalid* sequence, just to show the difference.